packages/discord.js/src/structures/Interaction.js

```diff
@@ -225,6 +225,14 @@ class Interaction extends Base {
       ComponentType[this.componentType] === ComponentType.SelectMenu
     );
   }
+
+  /**
+   * Indicates whether this interaction can be replied to or not.
+   * @returns {boolean}
+   */
+  isRespondable() {
```

Comment:
Isn't the name slightly misleading as autocomplete interactions can be `respond`ed to?
Maybe `replyable` instead?